### PR TITLE
Remove .js imports from src dir

### PR DESCRIPTION
**Pull request by iantrich**

### CODE REVIEW

File: src/entrypoints/service-worker-hass.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ function initRouting() {
   // Get manifest and service worker from network.
   workbox.routing.registerRoute(
     new RegExp(
-      `${location.host}/(service_worker.js|service_worker_es5.js|manifest.json)`
+      `${location.host}/(service_worker|service_worker_es5|manifest.json)`
```
Comment (iantrich) [inline src/entrypoints/service-worker-hass.js]: Not sure about this one

File: src/html/authorize.html.template [hunk 2]
```diff
@@ -34,7 +34,7 @@
       <ha-authorize><p>Initializing</p></ha-authorize>
     </div>
     <% if (!latestBuild) { %>
-      <script src="/static/custom-elements-es5-adapter.js"></script>
+      <script src="/static/custom-elements-es5-adapter"></script>
```
Comment (iantrich) [inline src/html/authorize.html.template]: I'm not sure about these `.template` file modifications I made either. Things seem to run fine in my testing...?

File: src/util/register-service-worker.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 const serviceWorkerUrl =
-  __BUILD__ === "latest" ? "/service_worker.js" : "/service_worker_es5.js";
```
Comment (balloob) [inline src/util/register-service-worker.js]: These should be reverted. Only ones that are used in either `import` or `import(…)`, as those are resolved by webpack. Others are resolved by the browser and that won't work.

File: src/html/authorize.html.template (hunk 2 above)
Comment (balloob) [reply src/html/authorize.html.template]: Yeah revert them. 

When you tested locally, did you navigate to the es5 version of authorize ?

**Comment by iantrich** [comment]
Going to scale this back to just lovelace to start in a new PR

**Pull request closed without merging**